api.go

```diff
@@ -272,6 +272,10 @@ type Repository struct {
 	// separator, generally '#' or ';'.
 	LineFragmentTemplate string
 
+	// Perf optimization: priority is set when we load the shard. It corresponds to
+	// the value of "priority" stored in RawConfig.
+	priority float64
```

Comment:
Why is this unexported when everything else is exported?

Reply:
It doesn't need to be exported, it is just an implementation detail. If we export it, it will end up in the index on disk which is unnecessary because the information is alrady contained in RawConfig.

Reply:
But couldn't we move this data from RawConfig to here? That's what ended up doing with RepositoryID I believe.

Reply:
It is actually the same. We still have the id in RawConfig, because that is how we transport the information from indexserver to the builder, and then duplicate it on load/unmarshal. 

I think it makes sense to move the logic to set priority to the same place.

Reply:
> It doesn't need to be exported, it is just an implementation detail.

Meaning, this PR doesn't require it. It might be that we have to export it soon :-)